components/bl-list-component/src/index.js

```diff
@@ -0,0 +1,30 @@
+import { List } from "./list";
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function ListComponent({ component, elRef, eventHandlers }) {
+  const { classList, display, width, backgroundColor, type, listItems, color, fontSize, padding } = component;
+
+  const items = listItems || [];
+
+  const onClickHandler = (e, itemContent, itemChildren) => {
```

Comment:
onClickHandler vs onItemClick

what's true?

Reply:
Changed to onItemClick